tenacity/__init__.py

```diff
@@ -86,8 +86,8 @@ def wrapped_f(*args, **kw):
 class Retrying(object):
     """Retrying controller."""
 
-    def __init__(self, marshaller, sleep=time.sleep):
-        self.marshaller = marshaller
+    def __init__(self, sleep=time.sleep, **kwargs):
```

Comment:
That's good, but it kind of kills the documentation as we lose the argument list. Since it's the major exposure of the API, I think it'd be better to just duplicate the list with Marshaller.